stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -294,9 +294,7 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
          endif ! lndp block
       end if
 
-    endif initalize_stochastic_physics
-
-    if (GFS_Control%do_ca .and. is_initialized) then
+    if (GFS_Control%do_ca) then
```

Comment:
I suggest aligning this if block and the code underneath by shifting 2 spaces t the right.

Reply:
agree

Reply:
Agree as well. @lisa-bengtsson I'll do as @SMoorthi-emc suggested.

Reply:
Done,  also aligned the `endif` so that the indentation matches.